optuna/visualization/_param_importances.py

```diff
@@ -25,16 +19,8 @@
 
     from optuna.visualization._plotly_imports import go
 
-    Blues = plotly.colors.sequential.Blues
+    bar_color = plotly.colors.sequential.Blues[-4]
```

Comment:
I think making a variable here is no more needed, as we remove `_distribution_colors`.